src/main/java/teammates/common/util/Const.java

```diff
@@ -1042,14 +996,9 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String MASHUP = "/mashup.jsp";
     
         //View fragments
-        public static final String INSTRUCTOR_HEADER = "/jsp/instructorHeader.jsp";
-        public static final String STUDENT_HEADER = "/jsp/studentHeader.jsp";
-        public static final String ADMIN_HEADER = "/jsp/adminHeader.jsp";
         public static final String FOOTER = "/jsp/footer.jsp";
-        public static final String STATUS_MESSAGE = "/jsp/statusMessage.jsp";
```

Comment:
All of the above were made obsolete during the transition from JSP to JSTL (in particular, #4106). For `FOOTER`, #5573 will remove it.